### PR TITLE
Task/pma 64 animated dn d

### PR DESCRIPTION
**Pull request by cptShell**

### CODE REVIEW

**Review by Nazeroth — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -18,15 +18,16 @@
   "dependencies": {
     "@hookform/resolvers": "2.8.8",
     "@reduxjs/toolkit": "1.8.1",
+    "@typescript-eslint/parser": "^5.28.0",
```
Comment (liza-veis) [inline package.json]: Lock dependency

File: package.json [hunk 2]
```diff
@@ -18,15 +18,16 @@
...
     "clsx": "1.1.1",
     "dnd-core": "16.0.1",
     "i18next": "21.7.1",
     "i18next-browser-languagedetector": "6.1.4",
     "immutability-helper": "3.1.1",
     "joi": "17.6.0",
+    "lodash": "4.17.21",
```
Comment (liza-veis) [inline package.json]: Why do we need lodash?

File: src/components/board/components/task-link/task-link.tsx [hunk 3]
```diff
@@ -1,46 +1,37 @@
...
+import { FC, useState, MouseEvent } from 'react';
+import { TaskDto, UserDto } from '~/common/types/types';
 import { ConfirmationModal } from '~/components/common/confirmation-modal/confirmation-modal';
 import { Modal } from '~/components/common/modal/modal';
 import { Task } from '../task';
 import styles from './styles.module.scss';
 import bucketImg from '~/assets/images/delete-bucket.svg';
+import { useAppDispatch } from '~/hooks/hooks';
+import { task as taskActions } from '~/store/actions';
+import { Draggable } from 'react-beautiful-dnd';
```
Comment (liza-veis) [inline src/components/board/components/task-link/task-link.tsx]: Move library import to the top

File: src/components/board/helpers/move-column.helper.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { DraggableLocation } from 'react-beautiful-dnd';
+import { ColumnResponse, FullColumnDto } from '~/common/types/types';
+import update from 'immutability-helper';
```
Comment (liza-veis) [inline src/components/board/helpers/move-column.helper.ts:2]: Move library import to top

File: src/components/board/helpers/move-task.helper.ts [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            [target.index, 0, sourceTask],
+          ],
+        },
+      },
+    });
+  }
+
+  const indexes = [sourceIndex, targetIndex];
+
+  result = indexes.reduce((acc, index) => orderTasks(acc, index), result);
```
Comment (liza-veis) [inline src/components/board/helpers/move-task.helper.ts]: If we assign to result this in all cases, why do we need to assign something to it above? It looks like it does't make sense and we can just use const

File: src/components/board/helpers/order-tasks.helper.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { FullColumnDto } from '~/common/types/types';
+import update from 'immutability-helper';
```
Comment (liza-veis) [inline src/components/board/helpers/order-tasks.helper.ts:1]: Move library import to top

File: package.json [hunk 7]
```diff
@@ -36,7 +36,9 @@
   },
   "devDependencies": {
     "@ls-lint/ls-lint": "1.11.0",
+    "@types/lodash": "4.14.182",
```
Comment (liza-veis) [inline package.json]: Remove this too

**Review by liza-veis — APPROVED** [review]

**Pull request merged**